.github/workflows/theme_auto_label.yml

```
@@ -0,0 +1,138 @@
+name: Theme auto label
+
+on:
+  issues:
+    types: [labeled]
+
+
+jobs:
+  auto-label:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if label is "Source Request"
+        id: label_check
+        run: |
+          LABEL_NAME=$(jq --raw-output .label.name "$GITHUB_EVENT_PATH")
+          if [[ "$LABEL_NAME" == "Source Request" ]]; then
+            echo "source_request=true" >> $GITHUB_OUTPUT
+          else
+            echo "source_request=false" >> $GITHUB_OUTPUT
+            echo "# ❌ Skipping: label is not Source request" >> $GITHUB_STEP_SUMMARY
+          fi
```

```suggestion
    if: contains(github.event.issue.labels.*.name, 'Source Request')
    runs-on: ubuntu-latest
    steps:
```
https://docs.github.com/en/actions/learn-github-actions/expressions#example-using-an-object-filter

---

I've read it but my brain never caught it...